rosidl_typesupport_c/test/test_message_type_support_dispatch.cpp

```diff
@@ -13,9 +13,12 @@
 // limitations under the License.
 
 #include <gtest/gtest.h>
-#include <rcutils/testing/fault_injection.h>
+
+#include <cassert>
```

Comment:
nit, I think the <cassert> include is no longer needed.

Reply:
Fixed here and in test_service...